cmd/tk/util.go

```diff
@@ -22,12 +22,17 @@ func pageln(i ...interface{}) {
 
 	// get system pager, fallback to `more`
 	pager := os.Getenv("PAGER")
+	var args []string
 	if pager == "" {
-		pager = "more"
+		// --raw-control-chars	Honors colors from diff.
+		// --quit-if-one-screen Closer to the git experience.
+		// --no-init 			Don't clear the screen when exiting.
+		pager = "less"
```

Comment:
What about systems that do not have `less`? For example `debian:buster-slim` or `ubuntu:latest` do not have those, but have `more`

Reply:
I'm following what git does (https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration): 

> You can set it to more or to your favorite pager (by default, it’s less), 

More doesn't have these options, its basically not useful.  This code will still work, without pagination, due to the error handling below.

Reply:
👍